src/coreclr/nativeaot/Test.CoreLib/src/Test.CoreLib.csproj

```diff
@@ -4,6 +4,7 @@
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
     <TargetFramework>netstandard2.0</TargetFramework>
     <EnableLibraryImportGenerator>true</EnableLibraryImportGenerator>
+    <DefineConstants>$(DefineConstants);TEST_CORELIB</DefineConstants>
```

Comment:
I'm saddened to see Test.CoreLib leaking out into shipping files.

Why do we need `<EnableLibraryImportGenerator>` in the first place? I see that if I remove that line, it generates errors in files under Runtime.Base, but Runtime.Base is supposed to be super low level and all DllImports in it were blittable. Why did we convert them to LibraryImport? Can we keep them as DllImport?

Reply:
We were generally following a rule of "Use source-generated interop everywhere". We can switch Test.CoreLib back to using DllImport. I'll add a comment about it so we don't forget why we added it.